adr/ADR-32.md

```diff
@@ -153,7 +153,14 @@ Returns a JSON having the following structure:
 ```typescript
 // EndpointInfo
 {
+    /**
+     * The name of the endpoint
+     */
     name: string,
+    /**
+     * The subject on which the endpoint is registered.
```

Comment:
I would simply say, the subject where the endpoint is listening. - how you got there doesn't matter.

Reply:
fixed